### PR TITLE
Update APICompat baseline version to 8.0.0

### PR DESCRIPTION
**Pull request by ViktorHofer**

### CODE REVIEW

**Review by carlossanlop — APPROVED** [review]

File: src/libraries/apicompat/ApiCompatBaseline.NetCoreAppLatestStable.xml [hunk 1]
```diff
@@ -1,22 +1,4 @@
 ﻿<?xml version="1.0" encoding="utf-8"?>
 <!-- https://learn.microsoft.com/en-us/dotnet/fundamentals/package-validation/diagnostic-ids -->
 <Suppressions xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xmlns:xsd="http://www.w3.org/2001/XMLSchema">
-  <Suppression>
```
Comment (carlossanlop) [inline src/libraries/apicompat/ApiCompatBaseline.NetCoreAppLatestStable.xml:4]: Why did you decide to remove these?

Comment (ViktorHofer) [reply src/libraries/apicompat/ApiCompatBaseline.NetCoreAppLatestStable.xml:4]: These suppressions were flagged as unnecessary now as the `RequiresPreviewFeaturesAttribute` attribute was removed from `RefreshMemoryLimit` between RC1 and GA.

Comment (carlossanlop) [reply src/libraries/apicompat/ApiCompatBaseline.NetCoreAppLatestStable.xml:4]: Ah that's neat. You are also told when a suppression is unnecessary. Cool.

Comment (ViktorHofer) [reply src/libraries/apicompat/ApiCompatBaseline.NetCoreAppLatestStable.xml:4]: Yes, that's a new feature in APICompat, part of .NET 8.

**Pull request merged**